packages/paymasters/contracts/AcceptEverythingPaymaster.sol

```diff
@@ -0,0 +1,41 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.6;
+pragma experimental ABIEncoderV2;
```

Comment:
nitpic: its `pragma abicoder v2;`
(not critical, since I think we'll have to release 0.8 contracts soon...)